listenbrainz_spark/ftp/download.py

```diff
@@ -145,9 +147,9 @@ def download_listens(self, directory, listens_dump_id=None, dump_type=FULL):
                 listens_file_name (str): name of downloaded listens dump.
                 dump_id (int): Unique indentifier of downloaded listens dump.
         """
-        ftp_cwd = current_app.config['FTP_LISTENS_DIR'] + 'fullexport/'
+        ftp_cwd = config.FTP_LISTENS_DIR + 'fullexport/'
```

Comment:
should use os.path.join to prevent issues with forgetting to add a / in the config file